cli_tools/gce_ovf_import/main.go

```diff
@@ -63,6 +63,7 @@ var (
 	scratchBucketGcsPath        = flag.String("scratch-bucket-gcs-path", "", "GCS scratch bucket to use, overrides what is set in workflow")
 	oauth                       = flag.String("oauth", "", "path to oauth json file, overrides what is set in workflow")
 	ce                          = flag.String("compute-endpoint-override", "", "API endpoint to override default")
+	computeServiceAccount       = flag.String("compute-service-account", "", "Compute service account to be used by importer Virtual Machine and the resulting VM or Machine Image. When empty, the Compute Engine default service account is used.")
```

Comment:
Added trimming.

Regarding validation, they can be emails but they can also be `default`. Maybe other IDs are acceptable as well, e.g. without the domain. 

I don't see much in that docs to be able to build a robust validation that we can trust it won't case false negatives. I suggest we leave it as is, since instance creation will do the validation for us, though a bit delayed.

Reply:
sgtm -- when I did a quick search I didn't see an authoritative answer on format either.